.eslintrc.js

```diff
@@ -0,0 +1,130 @@
+const TSEslint = require("@typescript-eslint/eslint-plugin")
+
+module.exports = {
+  parser: "babel-eslint",
+  extends: [
+    "google",
+    "eslint:recommended",
+    "plugin:flowtype/recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "prettier/flowtype",
+    "prettier/react",
+  ],
+  plugins: ["flowtype", "prettier", "react", "filenames"],
+  parserOptions: {
+    ecmaVersion: 2016,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    jest: true,
+  },
+  globals: {
+    before: true,
+    after: true,
+    spyOn: true,
+    __PATH_PREFIX__: true,
+    __BASE_PATH__: true,
+    __ASSET_PREFIX__: true,
+  },
+  rules: {
+    "arrow-body-style": [
+      "error",
+      "as-needed",
+      { requireReturnForObjectLiteral: true },
+    ],
+    "no-unused-expressions": [
+      "error",
+      {
+        allowTaggedTemplates: true,
+      },
+    ],
+    "consistent-return": ["error"],
+    "filenames/match-regex": ["error", "^[a-z-\\d\\.]+$", true],
+    "no-console": "off",
+    "no-inner-declarations": "off",
+    "prettier/prettier": "error",
+    quotes: ["error", "backtick"],
```

Comment:
We might wanna change this line to something I use in my TS repo with backticks:

https://github.com/LekoArts/gatsby-themes/blob/master/.eslintrc.js#L47-L54

Reasoning: https://github.com/typescript-eslint/typescript-eslint/pull/762

So ESLint otherwise wants to convert double quotes on types to backticks and the TS compiler doesn't like that at all!